pages/en/lb4/Decorator.md

```diff
@@ -0,0 +1,209 @@
+---
+lang: en
+title: 'Decorators'
+keywords: LoopBack 4.0, LoopBack-Next
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Decorators.html
+summary:
+---
+
+A decorator allows you annotate or modify your class declarations and members with metadata.
+
+## Introduction
+
+*If you're new to Decorators in TypeScript, see [here](https://www.typescriptlang.org/docs/handbook/decorators.html) for more info.*
+
+Decorators give LoopBack the flexibility to modify your plain TypeScript classes 
+and properties in a way that allows the framework to better understand how to 
+make use of them, without you needing to inherit base classes or add functions 
+that tie into an API.
+
+Currently, LoopBack core provides built-in decorator functions you can use directly
+by `@myDecorator`, e.g. Route decorators that introduces in section Route-Decorators, 
+and implements a dependency injection system that can apply a decorator by `@inject('myDecorator')`. 
```

Comment:
I don't think this is a valid usage: `@inject('myDecorator')`

`@inject` is a decorator for annotating dependencies to be injected, the parameter is the binding key that should be used to resolve the dependency. For example, `@inject('sequence.actions.findRoute')`.

Reply:
True, this part is from my old (wrong) understanding and I guess it also causes Kevin confused. Will remove it.